### DISCUSSION TITLE
Request for Help with NOAH 3.6 Model for Africa Morocco - Data Collection

### DISCUSSION THREAD

**Discussion by Mirylis** [NASA-LIS/LISF#1593] [Q&A]
Hello,

I'm a beginner user of the Land Information System (LIS), and I'm working with the NOAH 3.6 model to generate hydrological and energy balance outputs for a region in Morocco. However, I'm facing difficulties in finding and collecting the necessary data for this region.

Could someone guide me on where I can find open-source datasets that are compatible with LIS and suitable for Morocco? Any advice or resources would be greatly appreciated.

Thank you for your help!

Best regards,

**Comment by dmocko** [comment]
Hi @Mirylis

Two of the most commonly used global datasets are: 1) MERRA-2; and 2) GDAS

MERRA-2 data is available from the NASA GES DISC.  You need to download data for all of these 4 collections to use in LIS:
1. https://disc.gsfc.nasa.gov/datasets/M2T1NXSLV_5.12.4/summary
2. https://disc.gsfc.nasa.gov/datasets/M2T1NXFLX_5.12.4/summary
3. https://disc.gsfc.nasa.gov/datasets/M2T1NXLFO_5.12.4/summary
4. https://disc.gsfc.nasa.gov/datasets/M2T1NXRAD_5.12.4/summary

For each of the above 4 collections, there is one file for each day, with 24 hours contained within each file.

If you decide to try to use MERRA-2, please follow-up, as you need to organize the data in your directory in a specific way.

Another option is to use GDAS forcing: https://www.ncei.noaa.gov/products/weather-climate-models/global-data-assimilation

We have a long-time archive of LIS-ready GDAS data on our LIS Data Portal:
    https://portal.nccs.nasa.gov/lisdata_pub/data/MET_FORCING/GDAS

There are other options (PRINCETON, ERA5, etc.) but the LIS-ready data for these products are not as widely-available.
